src/core/packages/chrome/browser-internal/src/ui/header/header_top_banner.tsx

```diff
@@ -11,20 +11,56 @@ import React, { FC } from 'react';
 import useObservable from 'react-use/lib/useObservable';
 import { Observable } from 'rxjs';
 import type { ChromeUserBanner } from '@kbn/core-chrome-browser';
+import { css } from '@emotion/react';
 import { HeaderExtension } from './header_extension';
 
 export interface HeaderTopBannerProps {
   headerBanner$: Observable<ChromeUserBanner | undefined>;
+  position?: 'fixed' | 'static';
 }
 
-export const HeaderTopBanner: FC<HeaderTopBannerProps> = ({ headerBanner$ }) => {
+const styles = {
+  rootFixed: css`
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: var(--kbnHeaderBannerHeight);
+    width: 100%;
+  `,
+  rootStatic: css`
+    height: var(--kbnHeaderBannerHeight);
+    width: 100%;
+  `,
+  container: css`
+    height: 100%;
+    width: 100%;
+  `,
+};
+
+export const HeaderTopBanner: FC<HeaderTopBannerProps> = ({
+  headerBanner$,
+  position = 'fixed',
+}) => {
   const headerBanner = useObservable(headerBanner$, undefined);
   if (!headerBanner) {
     return null;
   }
 
   return (
-    <div className="header__topBanner" data-test-subj="headerTopBanner">
+    <div
+      css={[
+        position === 'static' ? styles.rootStatic : styles.rootFixed,
+        ({ euiTheme }) => ({
+          zIndex: euiTheme.levels.header,
+        }),
+        css`
+          .header__topBannerContainer {
```

Comment:
nit: I would consider incorporating `.header_topBannerContainer` with `styles.container` and just adding `styles.container` to the array above.